README.md

```diff
@@ -1,2 +1,36 @@
-# magento-payment-module
-Nelo Payment module for Magento
+# Nelo Payment Module for Magento
+Use the Nelo module for Magento and offer users to buy now and pay later.
+
+## How it works
+After install our module and enable it for your store with all required configuration, 
+the user will see Nelo promotional message as a payment option.  
+
+After user clics on "Continue to Nelo" button, he will be redirected to our web checkout
+and apply for a credit. Whether or not the credit is approved, when finished the user 
+will be redirected back to the corresponding page in your store.
+
+### Order states
+With our module when an order is placed the order is created with `pending_payment` state.
+If an error occurs during the process, the credit wasn't approved or the user cancels 
+the flow, the order will be set with `canceled` state.
+If all the process was successful the order will finish in `processing` state.
+
+## Requirements
+- PHP versions: 7.3 || 7.4
+- Magento versions: 2.3 || 2.4
+
+## Installation
+For now this module is provided as a Github repository. So you need to add 
```

Comment:
Therefore you need to add our repository (code below) to your `composer.json` in the repositories section.